src/CanRender.spec.tsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 
 import { CanRender, ElseRender, ElseCanRender } from '.';
 
 describe('CanRender#', () => {
   describe('when render with true condition', () => {
     it('should render children elements', async () => {
-      render(<CanRender when>test-can-render</CanRender>);
-      expect(await screen.queryByText('test-can-render')).not.toBeNull();
+      const { queryByText } = render(<CanRender when>test-can-render</CanRender>);
+
+      await waitFor(() => expect(queryByText('test-can-render')).not.toBeNull());
```

Comment:
What is the difference between old approach and this new?

Reply:
You are talking about the waitFor or the destructuring of queryByText from the render method?

Reply:
If it is about the destructuring, there is no difference.